lib/rspec/rails/adapters.rb

```diff
@@ -38,8 +38,7 @@ def self.disable_testunit_autorun
         gem 'minitest' if defined?(Kernel.gem)
         require 'minitest/unit'
         Assertions = MiniTest::Assertions
-      elsif ::Rails::VERSION::STRING >= '3.2.21'
-        # TODO: Change the above check to >= '3.2.22' once it's released
+      elsif ::Rails::VERSION::STRING >= '3.2.22'
```

Comment:
This was necessary for support for the newer Ruby 2.2 syntax. Officially I don't believe Rails 3.2.21 was supporting that just yet. There were some code and gem level changes they needed to do. So I think this was here just as a reminder about that condition and if we wanted to match 1:1 with the Rails support we would have to update the version string.

If you look at the Travis config for Rails [3.2.1](https://github.com/rails/rails/blob/v3.2.21/.travis.yml) and [3.2.2](https://github.com/rails/rails/blob/v3.2.22/.travis.yml) you can see in 3.2.2 they added Ruby 2.1 and 2.2.